normandy/settings.py

```diff
@@ -12,6 +12,7 @@ class Core(Configuration):
     INSTALLED_APPS = [
         'normandy.base.apps.BaseApp',
         'normandy.classifier',
+        'normandy.control.apps.ControlApp',
```

Comment:
`normandy.control.apps.ControlApp` is the newer style of installing Apps in Django. It should be preferred. classifier is using the old version because we were lazy and haven't yet needed any features provided by the newer version.